RELEASING.md

```diff
@@ -14,7 +14,14 @@ Create a new branch called `x.y.z-proposal` from the current commit.
 
 ## Use Lerna to prepare each package for release
 
-Use `npm install` and `lerna bootstrap` to initialize all package directories.
+Use below commands to initialize all package directories.
+
+```
+npm install
+
+lerna bootstrap
```

Comment:
Is the `lerna bootstrap` done automatically by the `npm install`, and if so can that step be removed?

Reply:
Makes sense, done